content/kots-cli/pull.md

```diff
@@ -18,32 +17,31 @@ kubectl kots pull [upstream uri] [flags]
 
 This command supports all [global flags](/kots-cli/global-flags/) and also:
 
+| Flag | Type | Description |
+|:-----|------|-------------|
+| `--downstream` | strings | the list of any downstreams to create/update |
+| `--exclude-admin-console` | bool | set to true to exclude the admin console _(only valid when `[upstream-uri]` points to a replicated app)_ |
+| `--exclude-kots-kinds` | bool | set to true to exclude rendering KOTS custom objects to the base directory _(default `true`)_ |
+| `-h, --help` | | help for pull |
+| `--helm-version` | string | the Helm version with which to render the Helm Chart _(default `"v2"`)_. This is a beta feature |
+| `--image-namespace` | string | the namespace/org in the docker registry to push images to _(required when `--rewrite-images` is set)_ |
+| `--license-file` | string | path to a license file _(required when `[upstream-uri]` points to a replicated app)_ |
+| `--local-path` | string | specify a local-path to pull a locally available replicated app _(only valid when `[upstream-uri]` points to a replicated app)_ |
+| `-n, --namespace` | string | namespace to render the upstream to in the base _(default `"default"`)_ |
+| `--registry-endpoint` | string | the endpoint of the local docker registry to use when pushing images _(required when `--rewrite-images` is set)_ |
+| `--repo` | string | repo uri to use when downloading a helm chart |
+| `--rewrite-images` | bool | set to true to force all container images to be rewritten and pushed to a local registry |
+| `--rootdir` | string | root directory that will be used to write the yaml to _(default `${HOME}` or `%USERPROFILE%`)_ |
+| `--set` | strings | values to pass to helm when running helm template |
+| `--shared-password` | string | shared password to use when deploying the admin console |
+| `--http-proxy` | string | sets HTTP_PROXY environment variable in all KOTS Admin Console components |
+| `--https-proxy` | string | sets HTTPS_PROXY environment variable in all KOTS Admin Console components |
+| `--no-proxy` | string | sets NO_PROXY environment variable in all KOTS Admin Console components |
+| `--copy-proxy-env` | bool | copy proxy environment variables from current environment into all KOTS Admin Console components |
+| `--config-values` | string | path to a manifest containing config values (must be apiVersion: kots.io/v1beta1, kind: ConfigValues) |
+| `--with-minio` | bool | set to true to include a local minio instance to be used for storage _(default true)_ |
 
-| Flag                      | Type    | Description                                                                                                                     |
```

Comment:
All the extra whitespace made this super unreadable imo

Reply:
Good catch, [the preview](https://deploy-preview-533--kotsio.netlify.app/kots-cli/admin-console/generate-manifests/) looks good